src/Simplex/Chat.hs

```diff
@@ -830,6 +848,21 @@ processChatCommand' vr = \case
                 else pure $ CRChatItemNotChanged user (AChatItem SCTGroup SMDSnd (GroupChat gInfo) ci)
             _ -> throwChatError CEInvalidChatItemUpdate
         CChatItem SMDRcv _ -> throwChatError CEInvalidChatItemUpdate
+    CTLocal -> do
+      (nf@NoteFolder {noteFolderId}, cci) <- withStore $ \db -> (,) <$> getNoteFolder db user chatId <*> getLocalChatItem db user chatId itemId
+      case cci of
+        CChatItem SMDRcv _ -> throwChatError CEInvalidChatItemUpdate
+        CChatItem SMDSnd ci@ChatItem {meta = CIMeta {editable}, content = ciContent} -> do
```

Comment:
doesn't really make sense to limit edit window in case of local items (see mkCIMeta)

Reply:
basically can just throw away check of `editable` from case